compiler/sem/evaltempl.nim

```diff
@@ -148,7 +154,13 @@ proc evalTemplateArgs(n: PNode, s: PSym; conf: ConfigRef; fromHlo: bool): PNode
 
   # add any generic parameters
   for i in 1..genericParams:
-    result.add n[givenRegularParams + i]
+    let it = n[givenRegularParams + i]
+    
+    # xxx: propagate nkError
```

Comment:
`xxx:` here means `TODO`, correct? Same with `QUESTION` tag - it is not really obvious what the author conveyed.

Reply:
TODO I tend to use for things to do before the PR is done.

Reply:
I tend to use `TODO` for things in the PR and `xxx` for stuff outside the PR.